tests/performance-test/performance-test.sh

```diff
@@ -52,28 +95,67 @@ do
 done
 
 if [ "${TOOL}" = "tg" ]; then
-    MSGS=$((HOSTS * PLUGINS))
-    LENGTH_S=$((COUNT / INTERVAL))
-    CONFIG="\
-- metadata:
-    name: SAF Performance Test 1
-  spec:
-    value-lists: ${MSGS}
-    hosts: ${HOSTS}
-    plugins: ${PLUGINS}
-    interval: ${INTERVAL}
-    length: ${LENGTH_S}"
-
-    echo "${CONFIG}" > deploy/config/generated-test-configs.yml
-    cd deploy
-    export TG_CONFIGFILE=./config/generated-test-configs.yml
-    ./performance-test-tg.sh
-
+    echo "Collectd-tg not implemented. Try running with '-t tb' instead"
+    exit 2
 elif [ "${TOOL}" = "tb" ]; then
-    export COUNT HOSTS PLUGINS INTERVAL CONCURRENT
-    cd deploy
-    ./performance-test-tb.sh
-
+    :
 else
     usage
 fi
+
+
+delete
+get_sources
+create_grafana_datasources
+create_service_monitors
+
+STAGE="TARGET"
+while true; do
```

Comment:
I'm reviewing a state-machine based operator written in bash. Nice.

Reply:
Maybe we should turn it into an actual operator